phangsPipeline/casaVisRoutines.py

```diff
@@ -1359,17 +1420,17 @@ def build_mstransform_call(
 
     # If not supplied by the user, find which SPWs should be included
     # in the processing.
-    
+
     if spw is None:
         if restfreq_ghz is not None:
             spw = find_spws_for_line(
                 infile=infile, restfreq_ghz=restfreq_ghz,
-                vlow_kms=vlow_kms, vhigh_kms=vhigh_kms,
+                vlow_kms=vstart_kms, vhigh_kms=vstart_kms+vwidth_kms,
```

Comment:
Based on the names, it seems like `vhigh_kms` needs to be given when `require_full_line_coverage=True`, instead of only 1 channel range

Reply:
My apologies... Would you mind elaborate on this? I am not sure I understand the suggestion here...

Reply:
The next line includes the check whether the full velocity range is contained within the SPW. My concern is that changing `vhigh_kms` to `vstart_kms+vwidth_kms` negates that check.

Reply:
So I made this change simply because `vhigh_kms` is not defined anywhere before this line, and `vstart_kms+vwidth_kms` seemed like the only possible alternative to me... Maybe I didn't fully grasp the code logic?

Reply:
I *think* this is fine since the velocities are transformed through to frequencies and then compared to the SPW range. vhigh_kms should be == vstart_kms+vwidth_kms, so this seems OK to me

Reply:
Ok, I agree now. I was interpreting `vwidth_kms` as the width of 1 channel, but it's the total velocity width across the target. Thanks both!